### ISSUE TITLE
Keep sidebars collapsed

### ISSUE THREAD

**Issue by tirsocarranza** [LemmyNet/lemmy-ui#2139]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

Hi,
Can you add maybe a cookie setting to keep sidebars collapsed?
Basically once you collapse a sidebar, keep it that way even if you navigate to other feed pages or if you close the tab (or browser) and open Lemmy page again.
I belive this can be accomplished even if you are not logged in.

Sidebars on Feed: Trending Communities, Instance Information and Subscribed Communities.
Sidebars on Posts: Community Info, Rules.

Thanks!

**Comment by dessalines** [comment]
Not interested. Can re-open if someone wants to take this on.
